### ISSUE TITLE
Yum broken when RPM from this requirement with underscores.

### ISSUE THREAD

**Issue by mosquito** [aio-libs/aiohttp-cors#43]
Please never use underscores in python-packages names.

[PEP-008 conatins](https://www.python.org/dev/peps/pep-0008/#package-and-module-names)

> Python packages should also have short, all-lowercase names, although the use of underscores is discouraged.

We have a really problem. When our build system convert all dependencies to RPM (using [FPM](https://github.com/jordansissel/fpm)) yum is broken. We are can't use this package in our projects.

**Comment by rutsky** [comment]
@mosquito thanks for spotting this, I didn't know that part of PEP8.

This is indeed contradict with PEP8 advices, but this is how (at least some) packages in `aiohttp_*` already named (e.g. `aiohttp_admin`, `aiohttp_session`).
@asvetlov we can raise discussion about renaming them or distributing both with underscore and hyphen versions (some of `aiohttp` packages indeed called with hyphen: `aiohttp-riak`, `aiohttp-jrpc`).

We are not only one who used underscore in packages names and underscores are disallowed in package names in RedHat based and Debian based distros, so FPM at least tries to replace underscores with hyphen: https://github.com/jordansissel/fpm/issues/65 , but looks like there are bugs in FPM with that: https://github.com/jordansissel/fpm/issues/1201 , https://github.com/jordansissel/fpm/issues/1016

Bottom line: at this point this is a bug of FPM (so in order to use `aiohttp_cors` with FPM you need to fix FPM), but also we should consider renaming `aiohttp_*` to `aiohttp-*` to line up with PEP8.

**Comment by mosquito** [comment]
Actually you should see how PyPi and PIP works. When you try to fetch packages which contains underscores in names PYPI will return redirect.

```
$ curl -I 'https://pypi.python.org/simple/aiohttp_cors/'
HTTP/1.1 301 Moved Permanently
Location: /simple/aiohttp-cors
...
```

```
$ curl -I 'https://pypi.python.org/simple/aiohttp-cors/'
HTTP/1.1 200 OK
Server: nginx/1.10.1
Content-Type: text/html; charset=utf-8
Content-Length: 1864
Date: Thu, 20 Oct 2016 10:22:42 GMT
...
```

But PyPi XML-RPC is "underscore sensitive". And you can't find this by `pip search aiohttp-cors`

When I developed [my PyPI server](https://github.com/mosquito/pypi-server) this caused some difficulties (have to search twice).

**Comment by rutsky** [comment]
@mosquito if PyPI/pip/setptools make such conversions, does it mean that we can just rename packages on PyPI to have hyphen instead of underscore, and nothing will break?

Does it mean, that currently all `aiohttp_cors` users can specify in their dependencies that they require `aiohttp-cors` and it will still download `aiohttp_cors` and everything will work?

**Comment by asvetlov** [comment]
Let's rename the package -- looks like it is safe and backward compatible operation

**Comment by asvetlov** [comment]
Fixed by c34202e
